app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+from app.player import Player
+
+
+class Elf(Player, ABC):

```

Comment:
The `Elf` class should implement the abstract methods `get_rating` and `player_info` from the `Player` class to be a concrete class. Consider adding implementations for these methods.